pyqtgraph/graphicsItems/ROI.py

```diff
@@ -71,6 +71,10 @@ class ROI(GraphicsObject):
                      is generally not necessary to specify the parent.
     pen              (QPen or argument to pg.mkPen) The pen to use when drawing
                      the shape of the ROI.
+    handlePen        (QPen or argument to pg.mkpen) The pen to use when drawing
+                     the ROI handles.
+    hoverPen         (QPen or argument to pg.mkpen) The pen to use while the
+                     mouse is hovering over the ROI shape or its handles.
```

Comment:
Can we also have `handleHoverPen` for completeness? 

..or maybe it would be nicer to provide `Handle.setPen` and `Handle.setHoverPen` instead..  I'd be happy with either approach.